UnitTests/PaymentsCoreTests/APIClient_Tests.swift

```diff
@@ -3,206 +3,112 @@ import XCTest
-
-        mockURLSession.cannedJSONData = jsonResponse
-
-        mockURLSession.cannedURLResponse = HTTPURLResponse(
-            url: URL(string: "www.fake.com")!,
-            statusCode: 500,
-            httpVersion: "1",
-            headerFields: [:]
-        )
-
-        apiClient.fetch(endpoint: fakeRequest) { result, _ in
-            guard case .failure(.unknown) = result else {
-                XCTFail()
-                return
-            }
-            expect.fulfill()
-        }
-        waitForExpectations(timeout: 1)
+    
+    func testSend_setsAcceptLanguageHeaderToEN() {
+        let sut = APIClient(config: sandboxConfig, http: http)
+        
+        let apiRequest = APIRequest(method: .post, path: "/sample/url")
+        
+        sut.send(apiRequest) { _ in }
+        let urlRequest = http.lastSentRequest!
+        
+        XCTAssertEqual("en_US", urlRequest.value(forHTTPHeaderField: "Accept-Language"))
     }
+    
+    func testSend_setsAuthenticationHeaderToBasicAuthWithClientID() {
 
-    func testFetch_whenPayPalDebugHeader_returnsCorrelationID() {
-        let expect = expectation(description: "Callback invoked.")
-
-        mockURLSession.cannedURLResponse = HTTPURLResponse(
-            url: URL(string: "www.fake.com")!,
-            statusCode: 200,
-            httpVersion: "1",
-            headerFields: ["Paypal-Debug-Id": "fake-id"]
-        )
-
-        apiClient.fetch(endpoint: fakeRequest) { _, correlationID in
-            XCTAssertEqual(correlationID, "fake-id")
-            expect.fulfill()
-        }
-        waitForExpectations(timeout: 1)
+        let sut = APIClient(config: sandboxConfig, http: http)
+        
+        let apiRequest = APIRequest(method: .post, path: "/sample/url")
+        
+        sut.send(apiRequest) { _ in }
+        let urlRequest = http.lastSentRequest!
+        
+        XCTAssertEqual("Basic MTIzOg==", urlRequest.value(forHTTPHeaderField: "Authorization"))
     }
-
-    func testFetch_withNoURLRequest_returnsNoURLRequestError() {
-        let expect = expectation(description: "Callback invoked.")
-
-        // Mock request whose API object does not vend a URLRequest
-        let noURLRequest = FakeRequestNoURL()
-
-        apiClient.fetch(endpoint: noURLRequest) { result, _ in
-            switch result {
-            case .success(_):
-                XCTFail()
-            case .failure(let error):
-                XCTAssertNotNil(error)
-            }
+    
+    func testSend_forwardsHTTPResponseToCompletionHandler() {
+        let httpResponse = HTTPResponse(status: 123)
+        http.stubSend(with: httpResponse)
+        
+        let sut = APIClient(config: sandboxConfig, http: http)
+        
+        let apiRequest = APIRequest(method: .post, path: "/sample/url")
+        
+        let expect = expectation(description: "forwards http response to completion handler")
+        sut.send(apiRequest) { response in
+            XCTAssertEqual(123, response.status)
```

Comment:
`APIClient` tests become simpler since the networking logic has been moved into `HTTPClient`.